infra/modules/providers/azure/provider/README.md

```diff
@@ -0,0 +1,23 @@
+# Azure Provider
+
+## Summary
+
+This module will configure a provider and install required packages mentioned below, needed by terraform to deploy Azure infrastructure resources.
+
+## Packages
+
+### Azure Resource Manager (Version 1.23.0 or higher)
+
+Azure Resource Manager enables you to repeatedly deploy your app and have confidence your resources are deployed in a consistent state. In Cobalt, we use Azure Resource manager extensively for deployment of our resources using Terraform
+
+### Terraform (Version 0.11.13 or higher)
+
+Terraform enables you to safely and predictably create, change, and improve infrastructure. It is an open source tool that codifies APIs into declarative configuration files that can be shared amongst team members, treated as code, edited, reviewed, and versioned. In Cobalt, we use Terraform for all our resource deployments on Azure.
+
+### Azure Active Directory
+
+Azure Active Directory provides reliability and scalability one needs with identity services that work with on-premises, cloud, or hybrid environment. In Cobalt, Azure Active directory is used for the traffic manager role assignment.
```

Comment:
Is this statement `Azure Active directory is used for the traffic manager role assignment.` accurate? Do we know for sure that the provider will only be used for traffic manager?

Reply:
@erikschlegel: I believe we can remove Azure Active directory altogether based on the comments above since, currently in cobalt, we are not using AzureAD anywhere. But since bedrock was using it, I had to include it. 
When we reference modules from bedrock, will they be using the `bedrock-provider` or will it use the `cobalt-provider` as in that case both providers will be different from each other? This is what I mentioned in the call yesterday that needs some clarity. 
According to this example: https://github.com/Microsoft/bedrock/blob/master/cluster/environments/azure-simple
If Azure AD is not part of the `cobalt-provider`, deployments which depend on Azure AD are going to fail.